Gems/AWSCore/Code/Include/Framework/AWSApiRequestJob.h

```diff
@@ -295,56 +268,86 @@ namespace AWSCore
         /// Called when request has completed successfully.
         virtual void OnSuccess()
         {
-            if (m_queueOnSuccess)
-            {
-                AZStd::function<void()> callbackHandler = [this]()
-                {
-                    if (m_onSuccess)
-                    {
-                        m_onSuccess(this);
-                    }
-                    delete this;
-                };
-                AZ::TickBus::QueueFunction(callbackHandler);
-            }
         }
 
         /// Called when the request fails.
         virtual void OnFailure()
         {
-            if (m_queueOnFailure)
-            {
-                AZStd::function<void()> callbackHandler = [this]()
-                {
-                    if (m_onFailure)
-                    {
-                        m_onFailure(this);
-                    }
-                    delete this;
-                };
-                AZ::TickBus::QueueFunction(callbackHandler);
-            }
         }
 
         /// Called when request can't process and still requires cleanup (Specifically for our derived class Function which does not use auto delete)
```

Comment:
nit: can we remove personal pronouns in the comment. I left them there, but we should be clear who the 'we' or 'our' is. Also is comment valid on base class now?